api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -43,6 +43,11 @@ message GrpcService {
     string authority = 2
         [(validate.rules).string =
              {min_len: 0 max_bytes: 16384 well_known_regex: HTTP_HEADER_VALUE strict: false}];
+
+    // Indicates the retry policy for re-establishing the gRPC channel
+    // This field is optional. If max interval is not provided, it will be set to ten times the provided base interval.
+    // Currently only supported for xDS gRPC streams.
```

Comment:
Please also state the defaults of the xDS gRPC streams.

Reply:
done